utils/build/docker/python/flask/app.py

```diff
@@ -175,6 +176,55 @@ def dbm():
     return Response(f"Integration is not supported: {integration}", 406)
 
 
+@app.route("/apm")
+def apm():
+    """
+        The goal of this endpoint is to trigger kafka integration calls
+        Usage:
+        - /apm?integration=kafka&applicationtype=producer
+        - /apm?integration=kafka&applicationtype=consumer
+    """
+    integration = flask_request.args.get("integration")
+    application_type = flask_request.args.get("applicationtype")
+    message_topic = "DistributedTracing"
+
+    request_action = {"kafka": {"consumer": kafka_consumer, "producer": kafka_producer,}}
+
+    action = request_action[integration][application_type]
+    action(message_topic)
+
+    return Response("OK")
```

Comment:
I'm not sure if there's a contract here or not.  I'd prefer to return an error if we time out or have an invalid invocation (i.e. the application type or integration are invalid), but it depends on what the system test framework expects.